scripts/workers/submission_worker.py

```diff
@@ -277,6 +279,15 @@ def extract_challenge_data(challenge, phases):
         evaluation_script_url, challenge_zip_file, challenge_data_directory
     )
 
+    try:
+        requirements_location = join(challenge_data_directory, "requirements.txt")
+        if os.path.isfile(requirements_location):
+            subprocess.check_output([sys.executable, "-m", "pip", "install", "-r", requirements_location])
+        else:
+            logger.info("No requirements for challenge {}".format(challenge.id))
```

Comment:
`No requirements for challenge {}".format(challenge.id)` -> `No custom requirements for challenge {}".format(challenge.id)`